CHANGES.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- More careful check for wrong parameter names being passed to NeuralNet
```

Comment:
Reference this PR or issue. (I have found reference the PR is more useful, since it usually also links to the issue)

Reply:
True. Added it.